docs/usage/security-and-permissions.md

```diff
@@ -65,6 +65,107 @@ For example, if you have an `npm` package and do not configure a private registr
 You could avoid this by configuring private registries but such registries need to query public registries anyway.
 We don't know of any public registries which reverse lookup IP addresses to associate companies with packages.
 
+#### Security awareness for self-hosted Renovate instances
+
+##### Introduction
+
+Before you start self-hosting Renovate you must understand the security implications associated with monitoring and updating repositories.
+The process that Renovate uses to update dependencies runs under the same user context as the Renovate process itself.
+This also means the process has the same level of access to information and resources as the user context!
+
+##### Trusting Repository Developers
+
+All self-hosted Renovate instances must operate under a trust relationship with the developers of the monitored repositories.
+This has the following implications:
+
+- Access to information
+- Execution of code
+
+Keep reading to learn more.
+
+###### Access to information
+
+Since the update process runs with the _same_ user privileges as the Renovate process, it inherently has access to the same information and resources.
+This includes sensitive data that may be stored within the environment where Renovate is hosted.
+
+###### Execution of code
+
+In certain scenarios, code from the monitored repository is executed as part of the update process.
+This is particularly true during, for example:
+
+- `postUpgradeTasks`, where scripts specified by the repository are run
+- when a wrapper within the repository is called, like `gradlew`
+
+These scripts can contain arbitrary code.
+This may pose a significant security risk if the repository's integrity is compromised, or if the repository maintainers have malicious intentions.
+
+Because such insider attack is an inherent and unavoidable risk, the Renovate project will not issue CVEs for such attacks or weaknesses other than in exceptional circumstances.
```

Comment:
```suggestion
Because such a/an insider attack is an inherent and unavoidable risk, the Renovate project will not issue CVEs for such attacks or weaknesses other than in exceptional circumstances.
```

I think we're missing a `a/an` between such and insider here?

Please edit the suggestion to the correct form, before applying it.